openqabot/openqa.py

```diff
@@ -18,6 +19,8 @@ def __bool__(self) -> bool:
         return self.url.netloc == "openqa.suse.de"
 
     def post_job(self, settings) -> None:
+        if 'CI_JOB_URL' in os.environ:
+            settings['__TRIGGERED_BY'] = os.environ['CI_JOB_URL']
```

Comment:
this will cause exception outside gitlab CI ,  but nice location to add

Reply:
Why will it cause an exception outside of the CI? As I've just mentioned, I've tested it using a simple script locally. I've also invoked it without having the env variable set and it worked (although of course then there's no URL added).

Reply:
`os.environ['CI_JOB_URL']` without defined `CI_JOB_URL` throws `KeyError` exception as is in reality python dict.

Reply:
ah, I,m blind ... you have it in condition :D I was focused on line 23 :(